src/generateJsonSchema.ts

```diff
@@ -48,7 +48,7 @@ export function generateJsonSchema(schema: FoxgloveMessageSchema): Record<string
         break;
       case "enum":
         fieldType = {
-          title: `${field.type.enum.name}: ${field.type.enum.description}`,
+          title: `foxglove.${field.type.enum.name}`,
```

Comment:
Why remove the enum description?

Reply:
Because it's not a title. Also I checked every instance of these and even though they had slightly different text from the `description` field, they were still redundant in every case.